scripts/dts/extract/clocks.py

```diff
@@ -153,6 +153,39 @@ def _extract_consumer(self, node_path, clocks, def_label):
                             clock_label,
                             prop_alias)
 
+                # If the provided clock has a fixed rate, extract its frequency
+                # as a macro generated for the clock consumer.
+                if clock_provider['props']['compatible'] == 'fixed-clock':
+                    clock_prop_name = 'clock-frequency'
+                    clock_prop_label = 'FIXED_CLOCK_FREQUENCY'
```

Comment:
want to change this to `CLOCKS_CLOCK_FREQUENCY`

Reply:
@anangl: What do you think about @galak suggestion?

Reply:
I'm fine with that.
I'm not sure if it makes it more clear what the label represents, but I don't have any better proposal.

Reply:
I'm wanting this so its consistent with the property name.

Reply:
Changed to `CLOCKS_CLOCK_FREQUENCY`.